### DISCUSSION TITLE
I am unable to login

### DISCUSSION THREAD

**Discussion by vrop-zo** [ente-io/ente#3444] [Q&A]
I have set up my account and trying to login it saying my password is incorrect even though my password was the same as I given before creation of account. And I don't have any recovery key to change the password. So pls help me

**Comment by Damariobros** [comment]
If you never retrieved and stored your recovery key at any point after creating your account, then there is no way to change your password from the outside, unfortunately.

Your password is directly used to encrypt your account's encryption keys. A copy of those keys are stored which are encrypted with the recovery key, which if supplied to Ente, would decrypt this copy, allowing you to encrypt them with a new password, which replaces the one with the lost password.

So without the recovery key, unless there's a problem with the system not correctly processing your password you give (which isn't impossible, given recent troubles people have had with their copy/pasted recovery keys not working), unfortunately the only thing you can do is keep trying with your password.

If you're absolutely sure you're entering the right password, perhaps it's possible you may be making a mistake entering it, or that you made a mistake entering it when creating your account. Or it could be the aforementioned possible problem with Ente itself.

Keep trying with your password, wait to see if there's an issue with the system that gets fixed, or contact support to get your account deleted so you can make a new one (so you don't have to make a new email address). These are the only pieces of advice I can give, I'm afraid. I'm sorry :(

In the end, whether you manage to regain access to your account, or you create a new one, it would be wise to view and save your account's recovery key to a secure location. You may print it out and store it in a safe, secure location, or you may copy/paste it to a text file, and save it on your computer well encrypted, to an external drive or memory card, and/or on a cloud service (preferably NOT one which you need Ente Auth to access --- a 'chicken and the egg' situation). Or do both, a little redundancy never hurts.
